### PR TITLE
Updating NetheriteProviderStartup to implement IWebJobsStartup

### PR DESCRIPTION
**Pull request by surgupta-msft**
Updating NetheriteProviderStartup to implement IWebJobsStartup instead of FunctionsStartup

### CODE REVIEW

File: src/DurableTask.Netherite.AzureFunctions/NetheriteProviderStartup.cs [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
     {
-        public override void Configure(IFunctionsHostBuilder builder)
+        public void Configure(IWebJobsBuilder builder)
         {
 #if !NETCOREAPP2_2
             builder.Services.AddSingleton<IDurabilityProviderFactory, NetheriteProviderFactory>();
 #else
             builder.Services.AddSingleton<IDurabilityProviderFactory, NetheriteProviderPseudoFactory>();
 #endif
+            builder.AddDurableTask();
```
Comment (davidmrdavid) [inline src/DurableTask.Netherite.AzureFunctions/NetheriteProviderStartup.cs:25]: I'm not a blocking reviewer here, but I'm surprised about this change. Why is this here now?

Comment (surgupta-msft) [reply src/DurableTask.Netherite.AzureFunctions/NetheriteProviderStartup.cs:25]: I definitely have very limited knowledge about this code. I am just trying to follow pattern used here  https://github.com/Azure/azure-functions-durable-extension/blob/dev/src/WebJobs.Extensions.DurableTask/DurableTaskWebJobsStartup.cs. Please share all the corrections needed here and I will update the PR.
cc [sebastianburckhardt](https://github.com/sebastianburckhardt)

Comment (sebastianburckhardt) [reply src/DurableTask.Netherite.AzureFunctions/NetheriteProviderStartup.cs:25]: I think the `builder.AddDurableTask()` is superfluous here... I would expect this to already be taken care of by the existing startup code in the durable task extension.

File: src/DurableTask.Netherite.AzureFunctions/NetheriteProviderStartup.cs [hunk 2]
```diff
@@ -1,26 +1,28 @@
-﻿// Copyright (c) Microsoft Corporation.
+// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.Collections.Concurrent;
```
Comment (sebastianburckhardt) [inline src/DurableTask.Netherite.AzureFunctions/NetheriteProviderStartup.cs:4]: In this project we follow the convention of putting the using clauses inside the namespace, as it was the case before. This is obviously not really important, but I prefer it to be consistent across all files in the same project.

**Review by sebastianburckhardt — CHANGES_REQUESTED** [review]
See comments.

File: src/DurableTask.Netherite.AzureFunctions/NetheriteProviderStartup.cs (hunk 2 above)
Comment (surgupta-msft) [reply src/DurableTask.Netherite.AzureFunctions/NetheriteProviderStartup.cs]: Makes sense. Corrected the code here.

File: src/DurableTask.Netherite.AzureFunctions/NetheriteProviderStartup.cs (hunk 1 above)
Comment (surgupta-msft) [reply src/DurableTask.Netherite.AzureFunctions/NetheriteProviderStartup.cs]: Done.

**Review by sebastianburckhardt — APPROVED** [review]
Looks good to me.
